docs/core/diagnostics/debug-threadpool-starvation.md

```diff
@@ -112,32 +97,45 @@ dotnet-counters monitor -n DiagnosticScenarios
 Press p to pause, r to resume, q to quit.
     Status: Running
 
+Name Current Value
```

Comment:
Can you align the 'Current Value' column so that it visually matches what you'd see in the tool?

Reply:
Done